src/test/java/org/folio/rest/impl/UserTenantIT.java

```diff
@@ -77,6 +78,26 @@ void canDeleteAllUserTenants() {
     Assertions.assertEquals(0, collection2.getTotalRecords());
   }
 
+  @Test
+  void canUpdateUserTenant() {
+    UserTenantCollection collection = userTenantClient.getAllUsersTenants();
+    UserTenant userTenant = collection.getUserTenants().get(0);
+    userTenant.setEmail("Test");
+    userTenant.setPhoneNumber("1234");
+    sendAffiliationUpdatedEvent(List.of(userTenant));
+    new java.util.Timer().schedule(
+      new java.util.TimerTask() {
+        @Override
+        public void run() {
+          UserTenantCollection collection2 = userTenantClient.getAllUsersTenants();
+          Assertions.assertEquals("Test", collection2.getUserTenants().get(2).getEmail());
+          Assertions.assertEquals("1234", collection2.getUserTenants().get(2).getPhoneNumber());
+        }
+      },
+      10000
```

Comment:
For unit test 10 sec can be long, also please add asserts for mobilePhoneNumber and username